lib/shopping_cart.rb

```diff
@@ -0,0 +1,53 @@
+require './lib/product'
+
+class Shopping_cart
```

Comment:
Make sure you create exactly what we asked for - based on the interaction pattern, we want a `ShoppingCart`